gemrb/core/GameScript/GSUtils.cpp

```diff
@@ -514,19 +514,20 @@ void DisplayStringCore(Scriptable* const Sender, ieStrRef Strref, int flags, con
 	}
 
 	if (soundpath && soundpath[0] && !(flags & DS_SILENT)) {
-		ieDword speech = 0;
-		Point pos = Sender->Pos;
-		if (flags&DS_SPEECH) {
-			speech = GEM_SND_SPEECH;
+		ieDword soundFlags = GEM_SND_EFX;
+		Point pos;
+		if (flags & DS_SPEECH) {
+			soundFlags |= GEM_SND_SPEECH;
 		}
-		// disable position, but only for party
+
 		Actor* actor = Scriptable::As<Actor>(Sender);
-		if (!actor || actor->InParty ||
-			core->InCutSceneMode() || core->GetGameControl()->InDialog()) {
-			speech |= GEM_SND_RELATIVE;
-			pos.reset();
+		// Spatial unless PC, cutscene or dialog
+		if (actor && !actor->InParty && !core->InCutSceneMode() && !core->GetGameControl()->InDialog()) {
+			Point pos = Sender->Pos;
```

Comment:
Should be without Point, so it affects the outer pos.

Reply:
Thanks